1.17-alpine-fuzz/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.17-alpine
+
+RUN apk add --no-cache gcc musl-dev git bash
+RUN wget -O- -nv https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.38.0
+RUN mv /go/bin/golangci-lint /usr/local/bin/
+RUN wget -O /usr/local/bin/swagger "https://github.com/go-swagger/go-swagger/releases/download/v0.26.1/swagger_linux_amd64"
```

Comment:
Let's update to https://github.com/go-swagger/go-swagger/releases/tag/v0.29.0

Reply:
Updated the url